### PR TITLE
BlockHound used to approve reactivity

### PR DESCRIPTION
**Pull request by kptfh**

### CODE REVIEW

File: src/test/java/org/springframework/data/aerospike/BaseReactiveIntegrationTests.java [hunk 1]
```diff
@@ -17,4 +23,23 @@
         return reactiveTemplate.findById(id, type).block();
     }
 
+    @BeforeClass
+    public static void installBlockHound() {
+        BlockHound.install();
+    }
+
+    @Test(expected = RuntimeException.class)
+    public void shouldFailAsBlocking(){
```
Comment (Aloren) [inline src/test/java/org/springframework/data/aerospike/BaseReactiveIntegrationTests.java]: this test will run for all classes that extend `BaseReactiveIntegrationTests`. We want it to run only once

Comment (kptfh) [reply src/test/java/org/springframework/data/aerospike/BaseReactiveIntegrationTests.java]: Do we want to be sure that all tests run in 'detect-blocks' mode or just one?

Comment (Aloren) [reply src/test/java/org/springframework/data/aerospike/BaseReactiveIntegrationTests.java]: this test is needed to verify that blockhound is setup correctly. why do we need it to test multiple times?

Comment (kptfh) [reply src/test/java/org/springframework/data/aerospike/BaseReactiveIntegrationTests.java]: I guess that when you run some test you would like to be sure that it runs in 'detect-blocks' mode

Comment (kptfh) [reply src/test/java/org/springframework/data/aerospike/BaseReactiveIntegrationTests.java]: For example manually in some IDE

Comment (Aloren) [reply src/test/java/org/springframework/data/aerospike/BaseReactiveIntegrationTests.java]: ok

**Pull request merged**